spec/models/category_spec.rb

```diff
@@ -7,6 +7,12 @@
   it { is_expected.to validate_presence_of :user_id }
   it { is_expected.to validate_presence_of :name }
 
+  it do
+    is_expected.to validate_numericality_of(:default_slow_mode_seconds).is_greater_than(
+      0,
+    ).only_integer
```

Comment:
A bit quirky, but this is what syntax_tree wants. 🙂

Reply:
You can probably try to add explicit parenthesis to have a less awkward formatting